CHANGELOG.md

```diff
@@ -37,6 +36,8 @@
 - Added missing nodes to the node menu
 - Added *Collection Operations* node.
 - Added *Rotations* output to the *Particles Data* node.
+- Added Get Vertex Color Layer node.
+- Added Insert Vertex Color Layer node.
```

Comment:
Why did you add them to 2.1.5? They should go to the Unreleased section.

Reply:
Sorry :) Now they are in the Unreleased section under the Added category.